core/src/main/resources/org/apache/spark/ui/static/historypage-template.html

```diff
@@ -77,12 +77,12 @@
   <tbody>
   {{#applications}}
     <tr>
+      {{#attempts}}
```

Comment:
Does this need a 'closing tag' `{{/attempts}} below then?

Reply:
yes, `{{/attempts}}` is in Line 95.

Reply:
Ah right , this was moved not added